spring-cloud-gcp-samples/spring-cloud-spanner-r2dbc-samples/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <!-- Your own application should inherit from spring-boot-starter-parent -->
+    <artifactId>spring-cloud-gcp-samples</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>3.5.4-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>spring-cloud-spanner-r2dbc-samples</artifactId>
+  <name>Spring Framework on Google Cloud Code Sample - Spanner-r2dbc</name>
+
+  <properties>
+    <sonar.skip>true</sonar.skip>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>cloud-spanner-r2dbc</artifactId>
+        <version>${cloud-spanner-r2dbc.version}</version>
```

Comment:
Can we use `spring-cloud-gcp-dependencies` instead, like we do in other samples?

Reply:
done